libcodechecker/server/api/report_server.py

```diff
@@ -53,6 +53,51 @@ class CountFilter:
     RUN_HISTORY_TAG = 6
 
 
+class DBSession():
+
+    def __init__(self, session_maker):
+        self.__session = None
+        self.__session_maker = session_maker
+
+    def __enter__(self):
+        # create new session
+        self.__session = self.__session_maker()
+        return self.__session
+
+    def __exit__(self, exc_type, exc_val, exc_tb):
+        if self.__session:
+            self.__session.close()
+
+
+def ExceptionConverter(func):
```

Comment:
I think the name ExceptionConverter is too general. Maybe a name that also reflects that this should be used for thrift calls would be great.

Reply:
@Xazax-hun  `ExceptionToThriftRequestError` ?